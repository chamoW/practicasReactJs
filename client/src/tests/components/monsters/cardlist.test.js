import React from "react";
import { CardList } from "../../../componentes/monsters/card-list/card-list.component";
import { shallow, mount, render } from "enzyme";

test("CardList componente test", () => {
    const mockRobots = [
        {
            id: 1,
            name: "Wladimir",
            username: "chamow",
            email: "teste@dss.com",
        },
    ];
    expect(shallow(<CardList monsters={mockRobots} />)).toMatchSnapshot();
});
