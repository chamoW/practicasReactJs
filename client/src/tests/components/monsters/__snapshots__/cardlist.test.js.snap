// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList componente test 1`] = `ShallowWrapper {}`;
